### ISSUE TITLE
LESS Version?

### ISSUE THREAD

**Issue by TedGoas** [colourgarden/avalanche#9]
Apologies if you've already considered or this has already been asked, but have you considered making a LESS version of this at any point? I realize LESS plays second fiddle to SASS, but LESS is a little easier to get up and running on Windows.

Just curious, really. Thanks for open-sourcing the logic behind your grid system!

**Comment by colourgarden** [comment]
Hi Ted,

I actually tried to port Avalanche to LESS for a client project with mixed success. The main functionality holes were lack of a proper `@each` function (LESS Guards just don't really work) and no support for key-value pairs ('maps' in Sass).

Therefore, it wasn't possible to loop through the width values (`$av-widths`) to generate all the width classes and it also wasn't possible to loop through the breakpoints (`$av-breakpoints`) to create the responsive classes automatically either.

I ended up with a halfway-house solution where I had a LESS mixin which created the numerator/denominator classes ok but it had to be run manually for each set of widths. I've uploaded a [pared down version](https://github.com/colourgarden/avalanche/files/77181/avalanche-less.txt) if you wish to take a look.

Saying all of this, my knowledge of LESS certainly isn't the best so it may well be possible to get close to the Sass version of Avalanche using some LESS functionality I'm unaware of.

Thanks for your interest in Avalanche.
